src/analyzer.ts

```diff
@@ -106,23 +107,20 @@ export class Analyzer {
    * see a mixture of pre-fork and post-fork contents when you analyze with a
    * forked analyzer.
    *
-   * Note: this feature is experimental.
+   * Note: this feature is experimental. It may be removed without being
+   *     considered a breaking change, so check for its existence before calling
+   *     it.
    */
   _fork(options?: ForkOptions): Analyzer {
-    const context =
-        options ? this._context._fork(undefined, options) : this._context;
+    const context = this._context._fork(undefined, options);
     return new Analyzer(context);
   }
```

Comment:
In my head, the semantics of `_fork` are that you want the cache from the existing analyzer, but you want to break with it, so that you won't see any new results in it and it won't see any new results from you.

Maybe there's no use case for calling it no options though. I mean, unless you're going to change the UrlLoader or something why not just use the original analyzer?